app/gui/src/controller/searcher/component.rs

```diff
@@ -124,24 +124,20 @@ pub struct ModuleGroups {
 #[derive(Clone, CloneRef, Debug, Default)]
 pub struct List {
     all_components:        Rc<Vec<Component>>,
-    top_modules:           group::List,
-    top_modules_flattened: group::List,
+    top_modules:           group::AlphabeticalList,
+    top_modules_flattened: group::AlphabeticalList,
     module_groups:         Rc<HashMap<Id, ModuleGroups>>,
     filtered:              Rc<Cell<bool>>,
+    /// Groups of components to display in the "Favorites Data Science Tools" section of the
+    /// Component Browser.
```

Comment:
Please use Rust doc links to `ComponentBrowser` implementation, so when reading these docs we can click there to learn more aobut hwat component browser is.